renovate.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended"
+  ],
+  "packageRules": [
+    {
+      "enabled": false,
+      "packagePatterns": [
+        "*"
+      ]
+    },
```

Comment:
Question (❓): With this configuration, we want to disable all dependency updates, just to start with, right?

Reply:
Yes, we experiment with Renovate for a8c-only dependencies for now